inbm/fpm/inb-provision-cloud/main.go

```diff
@@ -38,6 +37,28 @@ func main() {
 	flag.Usage = usage
 	flag.Parse()
 
+	var uccClientIdFile string
+	var uccServerIdFile string
+	var cloudCredentialPublicDir string
+	var cloudCredentialSecretDir string
+
+	if runtime.GOOS == "windows" {
+		uccClientIdFile = "c:\\intel-manageability\\inbm\\etc\\ucc\\client_id"
```

Comment:
is this where UCC wants their server_id, client_id in Windows? Will they have a different path?